README.md

```diff
@@ -165,24 +165,25 @@ Take the following steps to launch applications from sdl core.
   6. Both applications should show up on the head unit for launching
   7. Select the other application, and you should see it launched and brought to the foreground on the phone
 
-### Test Covarage
-Used technologies :
-  * GCOV - is a test coverage program.
-  * LCOV - is a graphical front-end for GCC's coverage testing tool for gcov.
-  * codecov.io - is service for assembling code coverage and represent in form clear for reading.
-
-Excluded folders :
-  * "/usr/\*" - local libraries shouldn`t be covered by tests.
-  * "\*/test/\*" - we don`t need to cover tests.
-  * "\*/src/3rd\*" - open source libraries shouldn`t be covered by tests.
-
-Current test coverage :
-You can find it in : [Coverage report](https://codecov.io/gh/smartdevicelink/sdl_core/branch/develop)
-
-How to get Test Coverage localy :
+## Test Coverage
+### Used technologies
+  * GCOV - test coverage program.
+  * LCOV - graphical front-end for GCC's coverage testing tool for gcov.
+  * codecov.io - service for assembling code coverage and representing it in a clear for reading form.
+
+### Excluded folders
+_We test only sources written by us and we don`t need to test external sources(open source libraries)._
+  * '/usr/\*' - local libraries shouldn`t be covered by tests.
+  * '\*/test/\*' - we don`t need to cover tests.
+  * '\*/src/3rd\*' - open source libraries shouldn`t be covered by tests.
+
+### Current test coverage
+You can find it in [Coverage report](https://codecov.io/gh/smartdevicelink/sdl_core/branch/develop)
+
+### How to get Test Coverage locally
  1. Build project
```

Comment:
@Kozoriz Of course it's understood that project has to be built unit-tests. But probably it's worth to specify the way how to build it here. (I mean `1. Build project with Unit-tests enabled. Execute make -DBUILD_TESTS=on`)